### DISCUSSION TITLE
Heads lose their lore after placing and breaking

### DISCUSSION THREAD

**Discussion by Sikatsu** [Zrips/CMI#4051] [Enhancement]
**Description of issue:**

---

When you place a mobhead and you want to replace it somewhere else, the lore of the mobhead is removed and not persistent.

---

[19:16:42 INFO]: CMI: 8.7.6.4 BungeeCord SqLite
[19:16:42 INFO]: Server: Paper("dccaf59a0") 1.16.2-R0.1-SNAPSHOT

**Comment by Zrips** [comment]
Not a bug but lack of feature. Placed items loose general information about them, like lore. So there is no easy way to keep track of it. Solution would be to store every single placed head in a file and access its information when some one breaks it and reapply it which is not present currently

**Comment by Sikatsu** [comment]
Ah, it would be nice if that could be added though. The old system that I used didn't lose the lore and such ^^
